src/backend/setup.cfg

```diff
@@ -87,7 +87,7 @@ dev =
     pytest-cov==4.1.0
     pytest-django==4.5.2
     pytest==7.4.0
-    responses==0.23.1
+    responses<0.23.2 # pin until boto3 supports urllib3>=2
```

Comment:
You should also update the `renovate.json` file at the root of the repo to prevent renovate to update this package.